src/controllers/trading.status.controller.ts

```diff
@@ -1,6 +1,16 @@
 import { Request, Response } from "express";
 import { Templates } from "../types/template.paths";
+import { TASK_LIST_PATH } from "../types/page.urls";
+import { Session } from "@companieshouse/node-session-handler";
 
 export const get = (req: Request, res: Response) => {
   return res.render(Templates.TRADING_STATUS);
 };
+
+export const post = (req: Request, res: Response) => {
+  const session: Session = req.session as Session;
+  const signInInfo = session.data["signin_info"] as object;
+  const companyNumber = signInInfo["company_number"] as string;
```

Comment:
instead of all this. Why not just use the company number in the url?

Reply:
Was in two minds about this we have always taken it from the session before, but if people are happy for me to get this from the url it would be less code.